source/server-discovery-and-monitoring/tests/rs/use_setversion_without_electionid.yml

```diff
@@ -97,21 +97,21 @@ phases: [
         outcome: {
             servers: {
                 "a:27017": {
+                    type: "RSPrimary",
```

Comment:
If A is supposed to become the primary I think this test needs more work. The comment above says:
```
    # Delayed response from A, reporting its reelection. Its setVersion shows
    # the election preceded B's so we ignore it.
```

Reply:
Updated the comments to clarify what the test should be getting at